### PR TITLE
IPFS Image extension

### PR DESCRIPTION
**Pull request by philipandersson**
https://github.com/denotedxyz/denoted-issues/issues/1

### CODE REVIEW

File: apps/web/src/lib/tiptap/ipfs-image.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+export const extension = Node.create({
+  name: 'ipfs-image',
+
+  addOptions() {
+    return {
+      inline: false,
+      HTMLAttributes: {},
+    }
+  },
```
Comment (ericzakariasson) [inline apps/web/src/lib/tiptap/ipfs-image.tsx:138]: Will we ever change this?

File: apps/web/src/lib/tiptap/ipfs-image.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  if (isError) {
+    return (
+      <NodeViewWrapper as="span">
+        <DataPill query={fetchImage}>no data</DataPill>;
+      </NodeViewWrapper>
+    );
+  }
+
+  return (
+    <NodeViewWrapper as="span">
```
Comment (ericzakariasson) [inline apps/web/src/lib/tiptap/ipfs-image.tsx:103]: Maybe we want it `as="div"`?

File: apps/web/src/lib/tiptap/ipfs-image.tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+          alt={alt}
+          title={title}
+          className="inline-block"
+          style={{ filter: isLoading ? "blur(4px)" : "none" }}
+        />
+        {isLoading && (
+          <Loader2
+            width={32}
+            height={32}
+            style={{ top: "calc(50% - 16px)", left: "calc(50% - 16px)" }}
```
Comment (ericzakariasson) [inline apps/web/src/lib/tiptap/ipfs-image.tsx:121]: ✨ Can be replaces with `inset-x-1/2 -translate-x-1/2  -translate-y-1/2 `

**Review by ericzakariasson — APPROVED** [review]
Great work 👏

File: apps/web/src/lib/tiptap/ipfs-image.tsx (hunk 1 above)
Comment (philipandersson) [reply apps/web/src/lib/tiptap/ipfs-image.tsx:138]: Most likely not, this was copied from the `@tiptap/extension-image` package.

File: apps/web/src/lib/tiptap/ipfs-image.tsx (hunk 2 above)
Comment (philipandersson) [reply apps/web/src/lib/tiptap/ipfs-image.tsx:103]: Not for now, unless we want inline images.

**Pull request merged**